.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
-  "typescript.tsdk": "node_modules/typescript/lib"
-}
```

Comment:
I think this should be kept to avoid conflicts with globally installed TS versions